### ISSUE TITLE
 What would be the best thing to extend meteor command?

### ISSUE THREAD

**Issue by endotakashi1992** [meteor/meteor#5789]
I making meteor's deploy tool.it is using `meteor deploy` command.Where is meteor command shell file?
if possible,I hope to use making tool after meteor official update.

**Comment by rozzzly** [comment]
Check your `$PATH`.  What platform you running on?

**Comment by endotakashi1992** [comment]
@rozzzly I'm running Meteor on OSX.
I want to do that then `meteor deploy abc.meteor.com`,exec official meteor command.and when `meteor deploy root@ownserver.com`,exec my function to deploy my server.
Do you know how to do that?

**Comment by rozzzly** [comment]
I'm not sure I follow... Could you rephrase that?

**Comment by stubailo** [comment]
Currently, Meteor doesn't support custom commands. I'd suggest making a shell script in your app directory and calling that instead.

**Comment by endotakashi1992** [comment]
@rozzzly Thanks to ans.My english is not perfect.This issue was close.

**Comment by endotakashi1992** [comment]
@stubailo Are you core developer in Meteor? Then,I want to say something.

This issue was created by dissatisfaction to Galaxy.Because it did not support small application.
I had been expected that all app can deploy by single command like that `meteor deploy`.But Galaxy is not support small app. I will not expect you.Make it myself.

Good bye.

**Comment by stubailo** [comment]
Galaxy has a developer plan for small apps now, if that helps.

**Comment by endotakashi1992** [comment]
@stubailo Oh! It's great!! I’m looking forward to complete it.

**Comment by Slava** [comment]
@endotakashi1992 the `meteor build` command can output your app as a node bundle. You can upload it and use on any platform you want. If you really want to build your own deployment command, just build a bash script that uses the command to deploy to your own server.

**Comment by endotakashi1992** [comment]
@Slava +1
Thank you.
